sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -1600,6 +1680,10 @@ export class ShareFileClient extends StorageClient {
     listHandles(options?: FileListHandlesOptions): PagedAsyncIterableIterator<HandleItem, FileListHandlesResponse>;
     get name(): string;
     get path(): string;
+    rename(destinationPath: string, options?: FileRenameOptions): Promise<{
```

Comment:
So this is aligning other rename APIs? Although REST api is applied on dest and using x-ms-file-rename-source as param?

Reply:
Yes, datalake SDK is doing the same. I talked with .Net team about this before, and they are doing the same.